makefiles/pseudomodules.inc.mk

```diff
@@ -1,5 +1,7 @@
 PSEUDOMODULES += at_urc
 PSEUDOMODULES += auto_init_dhcpv6_client
+PSEUDOMODULES += auto_init_actuators_sensors_%
+PSEUDOMODULES += auto_init_default
```

Comment:
Can we use `auto_init_actuators_sensors` instead of `auto_init_default`, I think `default is too generic`